### PR TITLE
Ui window size

### PR DESCRIPTION
**Pull request by Wincenttjoi**
Removed guisettings inside commons/core. Reason being is low cohesion and high coupling. 
Fixed big of default window size setting.

### CODE REVIEW

File: src/test/java/seedu/address/storage/StorageManagerTest.java [hunk 1]
```diff
@@ -41,7 +40,7 @@ public void prefsReadSave() throws Exception {
          * More extensive testing of UserPref saving/reading is done in {@link JsonUserPrefsStorageTest} class.
          */
         UserPrefs original = new UserPrefs();
-        original.setGuiSettings(new GuiSettings(300, 600, 4, 6));
+        //original.setGuiSettings(new GuiSettings(300, 600, 4, 6));
```
Comment (Choo-Xing-Yu) [inline src/test/java/seedu/address/storage/StorageManagerTest.java:43]: maybe leaving commented code could get confusing in the future?

Comment (Choo-Xing-Yu) [reply src/test/java/seedu/address/storage/StorageManagerTest.java:43]: @Wincenttjoi

Comment (Choo-Xing-Yu) [inline src/test/java/seedu/address/storage/StorageManagerTest.java:43]: hi @Wincenttjoi

**Comment by bobbylaw** [comment]
LGTM

**Pull request closed without merging**